### PR TITLE
feat(install): refactor getting_started and installation_guide

### PR DESCRIPTION
**Pull request by baptistemesta**
* We do not say  anymore that the 'installer' is for evaluation mode
* Document how to use a manually installed elasticsearch with installer
* refactor getting started to have all steps to do to fully use BICI

Closes [ICI-1056](https://bonitasoft.atlassian.net/browse/ICI-1056)

### CODE REVIEW

File: md/getting_started.md [hunk 1]
```diff
@@ -14,34 +14,44 @@ The archive contains:
...
 2. Ensure you have Docker installed, using command `docker --version`
 3. Docker requires an active Internet connection to pull the Elasticsearch Docker image and to build the BICI application image
-4. Follow the installation guide "evaluation mode" instructions from the archive.
-
-### Production mode
-
-1. Ensure you have Java installed, using the command `java -version`
-2. Follow the installation guide "production mode" instructions.
+4. Unzip the application  `bici-application-<version>.zip`
+5. Run `bin/bonita-ici` (`bin/bonita-ici.bat` on Windows platform)
```
Comment (laurentleseigneur) [inline md/getting_started.md]:
```suggestion
5. Run `bin/bonita-ici` (`bin/bonita-ici.bat` on Windows platform). or `sudo bin/bonita-ici`, if docker is available only for root user
```

File: md/prerequisites.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ Software required for BICI Add-on:
 
 Notes:
 1. BICI can be executed on Java 8. Java 9 and above are not yet supported. 
-1. All Subscription editions are supported.
-1. Archive must be activated
-1. BICI has its own connection pool to connect to Bonita engine database. Ensure database has available connections 
+2. All Subscription editions are supported.
```
Comment (laurentleseigneur) [inline md/prerequisites.md:63]: you don't need to change numbers, markdown will display them properly using `1.`for all items

**Review by laurentleseigneur — COMMENTED** [review]
some minor comment, but LGTM

File: md/prerequisites.md (hunk 2 above)
Comment (baptistemesta) [reply md/prerequisites.md:63]: I know, it was done automatically from my editor. I think its ok to have directly the right number in the file

**Pull request merged**